src/qml/QFieldCloudPopup.qml

```diff
@@ -318,14 +327,14 @@ Popup {
             function onPushFinished(projectId, hasError, errorString) {
               transferError.hasError = hasError;
               if (transferError.visible) {
-                transferError.detailsText = errorString;
+                transferError.detailsText = transferError.humanizeError(errorString);
```

Comment:
OK here I would do something different. 

If we have a human-friendly error, let's use that as the transferError *titleText*. For e.g., on the quota message, we want people to immediately understand it's a fixable quota issue, not a mysterious network error.